js/components/navbar.js

```diff
@@ -2,6 +2,7 @@ import { urls , isParticipantDataDestroyed } from "../shared.js";
 
 export const userNavBar = (data) => {
```

Comment:
Although not related to this issue, I suggest change `data` to `response` (or something similar) for clarity, since it looks like a response from backend API call.

Reply:
can do